Makefile

```diff
@@ -489,22 +483,22 @@ build-capi-headless-all: capi-setup
 test: $(foreach compiler,$(compilers),test-$(compiler)) test-packages test-examples test-deprecated
 
 test-singlepass-native:
-	cargo test --release --tests $(compiler_features) --features "test-singlepass test-native"
+	cargo test --release --tests $(compiler_features) -- singlepass::native
 
 test-singlepass-jit:
-	cargo test --release --tests $(compiler_features) --features "test-singlepass test-jit"
+	cargo test --release --tests $(compiler_features) -- singlepass::jit
 
 test-cranelift-native:
-	cargo test --release --tests $(compiler_features) --features "test-cranelift test-native"
+	cargo test --release --tests $(compiler_features) -- cranelift::native
 
 test-cranelift-jit:
-	cargo test --release --tests $(compiler_features) --features "test-cranelift test-jit"
+	cargo test --release --tests $(compiler_features) -- cranelift::jit
 
 test-llvm-native:
-	cargo test --release --tests $(compiler_features) --features "test-llvm test-native"
+	cargo test --release --tests $(compiler_features) -- llvm::native
 
 test-llvm-jit:
-	cargo test --release --tests $(compiler_features) --features "test-llvm test-jit"
+	cargo test --release --tests $(compiler_features) -- llvm::jit
```

Comment:
Agreed, I'll preserve this commands in case they are useful. But when we do global testing `make test` will do it performantly!

Reply:
Actually, looking back at it is not that trivial. I'll try to do it but it might not work because of some implications on the Makefile. If it doesn't work we can do that in a PR apart